joomla/joomla-cms
# Fixed url of joomla update error help button

Pull Request for Issue #39769 

### Summary of Changes
Fixed/changed the url of the joomla update error help button


### Testing Instructions
(It doesn't update joomla, it generates an error instead)
1. In the backend navigate to System
2. Under "Update" click Joomla. You will see the view of: administrator/index.php?option=com_joomlaupdate
3. Click the button "Upload and Update"
4. Instead of selecting a regular zipped version on Joomla, select a picture of type .png
5. Set the check box "I'm aware that a backup before any update is highly recommended."
6. Click the button "Upload & Install"
7. Confirm the action, if needed and start the installation.
8. This should result in an error like:
![grafik](https://github.com/joomla/joomla-cms/assets/8353850/3f1272b0-20cd-44a2-afc9-9079e55c7e6b)
9. Click on the button "Get help with this error"

### Actual result BEFORE applying this Pull Request
Instead of showing a help page, there is an error:

404 - RuntimeException
Error fetching page from MediaWiki API: The page you specified doesn't exist.

=> url: https://help.joomla.org/proxy?keyref=Help43:JHELP_COMPONENTS_JOOMLA_UPDATE&lang=en


### Expected result AFTER applying this Pull Request
The following help/doc page should be visualized:

Joomla Update Problems

(the content of this page is under construction)

=> url: https://docs.joomla.org/Special:MyLanguage/J4.x:Joomla_Update_Problems


### Link to documentations
Please select:
- [x] Documentation link for docs.joomla.org: https://docs.joomla.org/Special:MyLanguage/J4.x:Joomla_Update_Problems
